t/OptionTest.php

```diff
@@ -3,7 +3,6 @@
  * Unit tests for the Option class
  *********************************/
 
-require_once "PHPUnit/Framework.php";
```

Comment:
without this line, tests don't seem to work at all.
however, this line is also showing its age.

I was able to get tests to run with phpuni by adding this change on top of your branch:

~~~
diff --git a/t/OptionTest.php b/t/OptionTest.php
index 30d4c33..615e715 100644
--- a/t/OptionTest.php
+++ b/t/OptionTest.php
@@ -3,11 +3,12 @@
  * Unit tests for the Option class
  *********************************/
 
+use PHPUnit\Framework\TestCase;
 require_once "../optparse.php";
 
 function create_Option($args) { $opt = new Option($args); }
 
-class OptionTest extends PHPUnit_Framework_TestCase {
+class OptionTest extends TestCase {
 
     var $opt;
 
~~~